### ISSUE TITLE
The badge on the right side of chevron in the listitem

### ISSUE THREAD

**Issue by ShenghuGong** [react-native-training/react-native-elements#352]
### Description
According to the [lists badges doc](https://react-native-training.github.io/react-native-elements/API/lists/#badges),The badge on the left side of chevron in the listitem.
But,The badge on the right side of chevron.

The view order like this:
avatar title chevron badge

`react-native-elements` version: `0.11.2`

### Reproduction Steps and Sample Code
my code like this:
`<ListItem`
`  roundAvatar`
`  titleLines={1}`
`  avatar={{ uri: userInfo.profileImg }}`
`  title={'Ivan'}`
`  onPress={onPress || log}`
`  badge={{`
`  value: 3,`
`  textStyle: { color: 'orange' },`
`  containerStyle: { marginRight: 0 }`
`  }}`
      `/>`

**Comment by binoy14** [comment]
@ShenghuGong Can you provide a screenshot or [snack.expo.io](https://snack.expo.io) demo. Can't seem to be able to recreate it

**Comment by giordy** [comment]
@binoy14 I confirm it happens to me too.
I'm using React Native 0.43 (can't upgrade to 0.44 unfortunately yet).
I tried to recreate an example here https://snack.expo.io/BkZTVwdxb but it does not seem to happen on Expo :\
You need to press the button to have the badges appear (I'm using the same pattern implemented in my app).

In my case it happens in Android Emulator (NExus 5 API 22) and device Nexus 4 API 23.
I'm using react-native-elements 0.12.0 and react-native-vector-icons 4.1.1.

This is what I am seeing.
![selezione_011](https://cloud.githubusercontent.com/assets/6016950/26104328/a4975112-3a3d-11e7-9044-bd7abe067535.png)

**Comment by giordy** [comment]
Not sure if related but the style settings on the badge seem to be ignored (you always get a black badge with white text)

**Comment by binoy14** [comment]
@giordy Thanks for confirming. I will try to debug this

**Comment by camillo777** [comment]
Hey, this snack is crashing the expo app on my phone... Anyone? 
Samsung Galaxy Note 4 
6.0.1

**Comment by giordy** [comment]
@camillo777 it crashes mine too in Android, however it works on the Expo app in iPhone. It's probably a bug specific to Android

**Comment by jjdp** [comment]
+1

**Comment by SKoscarrovira** [comment]
I'm having this problem too, but on an iPhone. Haven't tried this on Android yet.
@giordy the style settings work for me, but skipping the badge prefix... probably the documentation is wrong:
```
badge={{value: 0, containerStyle: { backgroundColor: 'red' }}}
```

**Comment by krosk93** [comment]
I made a PR for this. I think it should fix this bug. Can someone else confirm it? All you need to do to test it is to change the version of the package in the package.json file to:
```
"react-native-elements": "git://github.com/SKoscarrovira/react-native-elements.git#6a0af4279ebf6c800737cd63a50c4c43a90764c3"
```

**Comment by giordy** [comment]
@krosk93 I tested your PR, solved the issue in my case.
Thanks for taking the time to solve this :)

**Comment by Monte9** [comment]
@krosk93 thanks for the PR. I'll merge the PR soon. 

Also we will be doing a release this weekend. So the changes should be live with RNE `v0.12.3`

**Comment by oky1** [comment]
Same problem.  I glad you solved it.

**Comment by Monte9** [comment]
This fix will be released in `v0.12.3`
